### ISSUE TITLE
Enable AddGraph operation to be run through the UI

### ISSUE THREAD

**Issue by d47853** [gchq/gaffer-tools#625]
This includes creating a component which can build a schema.

**Comment by n3101** [comment]
The UI fails to meet our Accessibility policy. It is unmaintainable and will be removed shortly.
